src/components/10-atoms/progress-bar/index.js

```diff
@@ -0,0 +1,93 @@
+import { LitElement, html, css, unsafeCSS } from 'lit-element';
+import { classMap } from 'lit-html/directives/class-map';
+import { defineVersioned } from '../../../utils/component-versioning';
+import { applyDefaults } from '../../../utils/with-react';
+import styles from './index.scss';
+
+class AXAProgressBar extends LitElement {
+  static get tagName() {
+    return 'axa-progress-bar';
+  }
+
+  static get styles() {
+    return css`
+      ${unsafeCSS(styles)}
+    `;
+  }
+
+  static get properties() {
+    return {
+      small: { type: Boolean, defaultValue: false },
+      fullWidth: { type: Boolean, defaultValue: false },
+      value: { type: String },
+      max: { type: String },
+      text: { type: String },
+    };
+  }
+
+  constructor() {
+    super();
+    applyDefaults(this);
+  }
+
+  calculatePercantage() {
+    //checks if the max value is empty or if its not a number
```

Comment:
This comment is already outdated. :D That's why I hate comments and prefer named methods.